staging/src/k8s.io/controller-manager/pkg/features/kube_features.go

```diff
@@ -47,5 +55,6 @@ func SetupCurrentKubernetesSpecificFeatureGates(featuregates featuregate.Mutable
 // cloudPublicFeatureGates consists of cloud-specific feature keys.
 // To add a new feature, define a key for it at k8s.io/api/pkg/features and add it here.
 var cloudPublicFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
-	IPv6DualStack: {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
```

Comment:
this was removed from the core, @cheftako @andrewsykim can we remove it here too?

Reply:
I see it is removed in next commit, but let's give visibility to the sig-cloud-provider owners

Reply:
Can we do this feature gate clean up in a separate PR?

Reply:
+1

Reply:
Other than breaking these unrelated bits out, I am LGTM on this PR

Reply:
OK, I will remove the second commit in that case.

Reply:
FYI: https://github.com/kubernetes/kubernetes/pull/116255

Reply:
yeah, I came across this while adding a feature gate for a different KEP. 116255 is in the merge queue now